app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -1,12 +1,140 @@
 import React from 'react';
+import { getEventCounts } from './queries';
+import { Engine, Render, World, Bodies } from 'matter-js';
+import { graphql } from 'react-apollo';
+import githubPull from '../../../../assets/images/pr.png';
```

Comment:
That react-scripts 3.0 absolute path fix would be nice here, huh?

Reply:
might be easier to add a webpack alias for images

Reply:
you used webpack and easy in the same sentence lol.